### PR TITLE
add main function

### PR DESCRIPTION
**Pull request by majsylw**
divide code into main parts

### CODE REVIEW

File: hamnosys_dicts.py [hunk 1]
```diff
@@ -14,9 +14,9 @@
 }
 
 HandlocationFronalPlaneRL = {
-    "0": "",  # left to the left
-    "1": "",  # left
-    # "2" : "", #center
+    # "0" : "", #center
```
Comment (majsylw) [inline hamnosys_dicts.py:17]: I moved this value to the begining

File: hamnosys_dicts.py [hunk 2]
```diff
@@ -125,10 +125,10 @@
 }
 
 HandLocationDistanceDict = {
-    "0": "",
-    "1": "",
-    "2": "",
-    #   "3" : None
+    #   "0" : None
```
Comment (majsylw) [inline hamnosys_dicts.py:128]: I moved this value to the begining

File: parse-hamnosys.py [hunk 3]
```diff
@@ -11,612 +11,629 @@
...
+        "NONDominant - Handposition - TB2",
+        "NONDominant - Handposition - Distance2",
+        "ERROR"],
+        fill_value=0)
+    # change some default values for chosen columns
+    data["Dominant - Handshape - Baseform"] = 99
+    data["Dominant - Handposition - Extended finger direction"] = 99
+    data["Dominant - Handposition - Palm orientation"] = 99
+    #data["Dominant - Handposition - LR"] = 2
+    data["Dominant - Handposition - TB"] = 14
```
Comment (majsylw) [inline parse-hamnosys.py:127]: I don't know why here we have 14

File: parse-hamnosys.py [hunk 4]
```diff
@@ -11,612 +11,629 @@
...
-for index, row in data.iterrows():
-    for i in range(4):
-        char = row["Hamnosys_copy"][i: i + 1]
-        for key, value in HandLocationDistanceDict.items():
-            if char == value:
-                data.at[index, "Dominant - Handposition - Distance"] = key
-                data.at[index, "Hamnosys_copy"] = row["Hamnosys_copy"][1:]
+    for index, row in data.iterrows():
+        char = Hamnosys_copy[index][0]
+        if pd.to_numeric(data.at[index, "Dominant - Handposition - TB"]) != 99:
```
Comment (majsylw) [inline parse-hamnosys.py:601]: here we are checking "Dominant - Handposition - TB", but change value for "Dominant - Handposition - LR"

Comment (majsylw) [reply parse-hamnosys.py:601]: Additionaly also I am not sure if 99 can appear here

**Review by mplantykow — COMMENTED** [review]
pls ignore

File: hamnosys_dicts.py [hunk 5]
```diff
@@ -14,9 +14,9 @@
 }
 
 HandlocationFronalPlaneRL = {
```
Comment (mplantykow) [inline hamnosys_dicts.py:16]: I am not sure about havig this dict at all. In fact we do not use it.

File: hamnosys_dicts.py (hunk 2 above)
Comment (mplantykow) [reply hamnosys_dicts.py:128]: ok

File: parse-hamnosys.py (hunk 3 above)
Comment (mplantykow) [reply parse-hamnosys.py:127]: If not found then deafault is assumed, and default means 14. We can move 14 to 0 in dict and use 0.
So, if a sign that should determine Top/Bottom position of a hand/hands for given sign is not present in the notation, it is assumed that hands are placed in a typical/comfortable position.

File: parse-hamnosys.py (hunk 4 above)
Comment (mplantykow) [reply parse-hamnosys.py:601]: > here we are checking "Dominant - Handposition - TB", but change value for "Dominant - Handposition - LR"

Correct behavioru. LR sign is always placed around TB.
LRs:
 left to the left,
 left,
None
 right
 right to the right. 
Let's say  is TB.

In here we say "if you have found TB () before, and now you see  or .."

**Review by mplantykow — APPROVED** [review]
LGTM

**Pull request merged**